benchmarks/tpch_benchmark.py

```diff
@@ -5,8 +5,14 @@
 
 def get_valid_cases():
     result = [["query_id", "scale_factor", "format"]]
+    scale_factors = [1, 10]
+
+    machine_info = conbench.runner.machine_info(host_name=None)
+    if int(machine_info["memory_bytes"]) <= 64 * 1024 * 1024 * 1024:
+        scale_factors = [1]
```

Comment:
Could we make this be closer to 32GB? IIUC, we know that 64gb is just fine, but 32gb isn't sufficient. If we had some estimation of what the threshold where this falls over that would be nice to put here (and maybe * 1.1 so that we know it doesn't ever run there?) But 64gb seems too high, yeah?

Reply:
Yeah, I like 1.1 * so this never gets caught out by unexpected floating point error or whatever. (It shouldn't anyway, but still.) Updated to `1.1 * 32 * 1024 * 1024 * 1024`

Reply:
I also like the comments — do we know what the passing point is? Totally ok if we don't, but maybe adding that uncertainty would be nice to our future selves. In case we discover 48gb doesn't work as well

Reply:
I don't think we do. I wonder if we could track the memory consumption for query 21 on a machine with enough memory? For now I'll make it clear in the comment that we don't know